package.json

```diff
@@ -5,6 +5,7 @@
   "main": "plugin.js",
   "types": "index.d.ts",
   "scripts": {
+    "lint": "standard | standard",
```

Comment:
Sup with the double standard?

Reply:
what's wrong with double standards? /troll

Reply:
I get it less double standards, we need to be snazzy